contracts/src/v0.8/ccip/test/onRamp/OnRamp/OnRampWithMessageTransformer.setMessageTransformer.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {IMessageTransformer} from "../../../interfaces/IMessageTransformer.sol";
+import {OnRampWithMessageTransformer} from "../../../onRamp/OnRampWithMessageTransformer.sol";
+import {MessageTransformerHelper} from "../../helpers/MessageTransformerHelper.sol";
+import {OnRampSetup} from "./OnRampSetup.t.sol";
+
+contract OnRampWithMessageTransformer_setMessageTransformer is OnRampSetup {
+  OnRampWithMessageTransformer internal s_onRampWithMessageTransformer;
+  MessageTransformerHelper internal s_outboundMessageTransformer;
+
+  function setUp() public virtual override {
+    super.setUp();
+    s_outboundMessageTransformer = new MessageTransformerHelper();
+    s_onRampWithMessageTransformer = new OnRampWithMessageTransformer(
+      s_onRamp.getStaticConfig(),
+      s_onRamp.getDynamicConfig(),
+      _generateDestChainConfigArgs(s_sourceRouter),
+      address(s_outboundMessageTransformer)
+    );
+  }
+
+  function testSetMessageTransformer() public {
```

Comment:
wasn't able to add underscores (e.g. `test_setMessageTransformer`) in the name. getting `Function name must be in mixedCase  func-name-mixedcase` warning